src/Components/NewEmail/index.jsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from "react"
+import "./NewEmail.css"
+
+const NewEmail = ({closeNewEmail, setSentSuccess}) => {
+    const [error, setError] = useState(false)
+    const [address, setAddress] = useState('')
+    const [subject, setSubject] = useState('')
+    const [body, setBody] = useState('')
+    
+    const isValidEmail = (email) => {
+        return /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/g.test(email)
+    }
+
+    const handleAddress = (e) => {
+        if(!isValidEmail(e.target.value)) {
+            setError('Email is invalid')
+        } else {
+            setAddress(e.target.value)
+            setError(null)
+        }
+    }
+
+    const handleSubject = (e) => {
+        setSubject(e.target.value)
+    }
+
+    const handleBody = (e) => {
+        setBody(e.target.value)
+    }
+    
+    const sendEmail = async () => {
+        const sentEmail = {
+            email: address,
+            name: 'Colin & The Caterpillars',
+            subject: subject,
+            body: body
+        }
+        const response = await fetch (`http://localhost:8080/emails`, {
+            method: 'POST',
+            body: JSON.stringify(sentEmail),
+            headers: {
+                'content-type': 'application/json'
+            }
+        })
+        const emailResponseData = await response.json()
+        if (emailResponseData.data.sent) {
+            closeNewEmail()
+            setSentSuccess(true)
+        } else {
+            setSentSuccess(false)
+        }
+    }
+
+    return (
+        <div className="col-12 col-md-7 offset-md-2 offset-lg-1 col-lg-7 newEmail bg-white position-fixed border"> 
+            <div className="mb-3">      
+                <label htmlFor="to" className="form-label"></label>
```

Comment:
empty label tag? this is bad accessibility, screen readers will try and read this